scripts/verify-replace.sh

```diff
@@ -16,26 +20,57 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-error.XXX)
```

Comment:
let's keep some consistency with the case of the variables. it's fine to keep the local variables lower-case, but let's apply the rule to all of them.

Reply:
ahh , yes changed it now